### PR TITLE
progressBar completed

### PR DESCRIPTION
**Pull request by kxllytrxn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     "start": "next start"
   },
   "dependencies": {
+    "@next/swc-darwin-x64": "^12.3.1",
+    "@ramonak/react-progress-bar": "^5.0.2",
```
Comment (janiseliang) [inline package.json:11]: delete both of these lines? it doesn't seem like you're using the react-progress-bar i think

File: pages/order.js [hunk 2]
```diff
@@ -233,12 +246,14 @@ export default function Order() {
       </p>
     </div>
   </div>
+  console.log(formStep);
```
Comment (janiseliang) [inline pages/order.js:249]: delete

File: pages/order.js [hunk 3]
```diff
@@ -72,62 +73,74 @@ export default function Order() {
 
   /* title and navigation bar for orders */
   // TODO: add progress indicator
```
Comment (janiseliang) [inline pages/order.js:75]: this is done now - can delete the TODO

**Review by janiseliang — CHANGES_REQUESTED** [review]
looks good! just a few quick changes, and then we'll check with staff to get their opinion

**Review by justinazhang — APPROVED** [review]

**Review by justinazhang — APPROVED** [review]

**Pull request merged**